DER/DerivativesContracts/Options.rdf

```diff
@@ -189,57 +186,26 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The seller (issuer) of the call option assumes the obligation of delivering the assets specified should the buyer exercise the option. The buyer has the option, but not the obligation, to purchase the underlying asset.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;EquityOption">
-		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;EquityDerivative"/>
+	<owl:Class rdf:about="&fibo-der-der-opt;CombinationOption">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Option"/>
```

Comment:
Not sure the combination is itself an Option - it does not have a (single) strike price, expiration date, instrument identifier etc

Reply:
@rivettp This representation came out of our discussion in the last couple of DER meetings and with back and forth from John G and Jeff B.  We can review again, but that's where this came from.  There are other combination options that need to be subclasses of this class that we don't have covered yet, fyi.